### ISSUE TITLE
Create Gantt Chart with forecasted Sprints

### ISSUE THREAD

**Issue by holtzmak** [holtzmak/Humane-Transport#18]
This chart should be under revision control. Involve the Scrum Master and Product Owner in it's design.

The final product (for this issue at least) should be uploaded to GitHub.io.

**Comment by SanaKhan1997** [comment]
@holtzmak could you start looking into possible phases of our project e.g. prototyping, design documentation, etc. So that we can start planning gantt chart. I'm looking for a way to put the milestone planner i.e. gantt chart on our .io site.

**Comment by holtzmak** [comment]
A strong alternative to a Gantt Chart is a Activity Based Schedule - will be investigating both for a preferred alternative

**Comment by holtzmak** [comment]
I created the Gantt Chart to illustrate our Sprints and Sprint Goals as it provided a better visual illustration over the text-based Activity-Based Schedule. I ended up with the nest of both works by using the tasks of the Gantt Chart to illustrate activities.

Will not be putting the Gantt Chart under revision control as the document is forward-facing, meaning that we'll never undo the work represented on it. The chart can be found on our [website](https://sanakhan1997.github.io/Humane-Transport/) or the reviewable version is on [Google sheets](https://docs.google.com/spreadsheets/d/19Tyug4WldijKs0oYW3Ig37oQOJf8f5475Suza1lVvBs/edit?usp=sharing).
